pvlib/pvsystem.py

```diff
@@ -333,30 +336,32 @@ def get_aoi(self, solar_zenith, solar_azimuth):
 
     @_unwrap_single_value
     def get_irradiance(self, solar_zenith, solar_azimuth, dni, ghi, dhi,
-                       dni_extra=None, airmass=None, model='haydavies',
-                       **kwargs):
+                       albedo=None, dni_extra=None, airmass=None,
+                       model='haydavies', **kwargs):
         """
         Uses the :py:func:`irradiance.get_total_irradiance` function to
         calculate the plane of array irradiance components on a tilted
-        surface defined by ``self.surface_tilt``,
-        ``self.surface_azimuth``, and ``self.albedo``.
+        surface defined by ``self.surface_tilt`` and ``self.surface_azimuth```.
 
         Parameters
         ----------
-        solar_zenith : float or Series.
+        solar_zenith : float or Series
             Solar zenith angle.
-        solar_azimuth : float or Series.
+        solar_azimuth : float or Series
             Solar azimuth angle.
         dni : float or Series or tuple of float or Series
-            Direct Normal Irradiance
+            Direct Normal Irradiance. [W/m2]
         ghi : float or Series or tuple of float or Series
-            Global horizontal irradiance
+            Global horizontal irradiance. [W/m2]
         dhi : float or Series or tuple of float or Series
-            Diffuse horizontal irradiance
-        dni_extra : None, float or Series, default None
-            Extraterrestrial direct normal irradiance
+            Diffuse horizontal irradiance. [W/m2]
+        albedo : None, float or Series, default None
+            Ground surface albedo. [unitless]
+        dni_extra : None, float, Series or tuple of float or Series,
+            default None
```

Comment:
```suggestion
        dni_extra : None, float, Series or tuple of float or Series,\
            default None
```
Otherwise the `default None` gets counted as part of the parameter description, not the type.  Or could shorten the whole thing to `float, Series, or tuple of float or Series, optional`